.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         os-version: [ 20.04, 22.04 ]
```

Comment:
ubuntu-20.04 and macos-12 are deprecated.
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

I'm not sure if this is a case of raising the TEST OS version, as I've never done it before.

Reply:
I think deleting it would be the way to go for now.